File: Linguagem_Python/LG1_Lista1/L1C3.py
# Feito por Nícolas Henriques
# https://github.com/NicolasHenriques 
# Feito em 08/04/2022

print('--Azulejando Paredes--')
ap = float(input('Entre com a altura da parede: '))
lp = float(input('Entre com a largura da parede: '))
aa = float(input('Entre com a altura do azulejo: '))
ap = float(input('Entre com a largura do azulejo: '))

ta = (ap * lp) / (aa * ap)

print('O total de azulejos necessários serão: ', ta)

File: Linguagem_C/LG1_Lista1/L1C3.cpp
#include<stdio.h>
#include<conio.h>

int main()
{
	float aa, la, ap, lp, qa, arp, ara;
	
	printf("Entre com a altura da parede: ");
	scanf("%f", &ap);
	printf("Entre com a largura da parede: ");
	scanf("%f", &lp);
	printf("Entre com a altura do azulejo: ");
	scanf("%f", &al);
	printf("Entre com a largura do azulejo: ");
	scanf("%f", &la);

	arp = ap * lp;
	ara = aa * la;
	qa = arp / ara;
	
	printf("O numero de azulejos eh: %f", qa);
	
	getch();
	return 0;
}
